### ISSUE TITLE
undefined step?

### ISSUE THREAD

**Issue by xsuoownit** [cchacin/cucumber-common-steps#20]
I saw on your read me page that you support the following step:
> When I make a GET call to "/zen" endpoint in host "https://api.github.com"

But I don't see it defined anywhere. Is it defined?

**Comment by cchacin** [comment]
@xsuoownit 
That regex was removed, this one cover the case:

```gherkin
When I make a GET call to "https://api.github.com/zen" endpoint
```

**Comment by cchacin** [comment]
Fixed README c89c93b0518e31adbe338cf6962b01994077946a
